Description: Remove git ls-files command from gemspec
Author: Lucas Albuquerque Medeiros de Moura <lucas.moura128@gmail.com>
Last-updated: 2016-07-15
Forwarded: not-needed 
Bug: not-needed

--- a/em-spec.gemspec
+++ b/em-spec.gemspec
@@ -10,11 +10,11 @@ Gem::Specification.new do |s|
   s.summary = "BDD for Ruby/EventMachine"
   s.description = "Simple BDD API for testing asynchronous Ruby/EventMachine code"
   s.email = %q{schmurfy@gmail.com}
-  s.files = `git ls-files`.split("\n")
+  s.files = Dir.glob('**/*')
   s.homepage = %q{https://github.com/joshbuddy/em-spec}
   s.require_paths = ["lib"]
   s.rubygems_version = %q{1.3.7}
-  s.test_files = `git ls-files`.split("\n").select{|f| f =~ /^test/}
+  s.test_files = Dir.glob('**/*').select{|f| f =~ /^test/}
   s.rubyforge_project = 'em-spec'
 
   s.add_dependency 'eventmachine'
